broker-cli/proto/broker.proto

```diff
@@ -557,7 +579,12 @@ service OrderService {
    * └─────────────────────────────────────────────┴───────┴──────────────────┴──────────────────┴──────┴──────────┘
    * ```
    */
-  rpc GetBlockOrders (GetBlockOrdersRequest) returns (GetBlockOrdersResponse);
+  rpc GetBlockOrders (GetBlockOrdersRequest) returns (GetBlockOrdersResponse) {
+    option (google.api.http) = {
+      get: "/v1/orders"
```

Comment:
have we tested the `market` param here? example: `/v1/orders?market=btc/ltc`

Reply:
yes 👍